clusterloader2/testing/load/config.yaml

```diff
@@ -19,6 +19,7 @@
 {{$ENABLE_PROMETHEUS_API_RESPONSIVENESS := DefaultParam .ENABLE_PROMETHEUS_API_RESPONSIVENESS false}}
 {{$ENABLE_PVS := DefaultParam .CL2_ENABLE_PVS true}}
 {{$ENABLE_NETWORKPOLICIES := DefaultParam .CL2_ENABLE_NETWORKPOLICIES false}}
+{{$ENABLE_DENSITY_TEST := DefaultParam .CL2_DENSITY_TEST false}}
```

Comment:
I think CL2_ENABLE_DENSITY_TEST is better variable name: it avoids unnecessary renaming.

Reply:
typo, thanks for noticing.